packages/former/vite.config.playground.ts

```diff
@@ -4,6 +4,7 @@ import { defineConfig } from 'vite';
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  base: '',
```

Comment:
@lukashass No need to hardcode it in the vite config. Just run `pnpm run build:playground --base /former/` in the playground.yml insteadof just `pnpm run build:playground`.

Reply:
:+1: